### ISSUE TITLE
UMUI_UNPAGE_SERIALNUMBER failure

### ISSUE THREAD

**Issue by Nutzzz** [SuperPat45/UltraModernUI#2]
UMUI_UNPAGE_SERIALNUMBER doesn't appear to work.  Could I be doing something wrong?

```
Call must not be used with functions starting with "un." in the non-uninstall sections.
Usage: Call function_name | [:label_name]
Error in macro MUI_INSTALLOPTIONS_WRITE_DEFAULTCONVERT on macroline 14
Error in macro UMUI_FUNCTION_SERIALNUMBERPAGE on macroline 16
Error in macro UMUI_PAGE_SERIALNUMBER on macroline 35
Error in macro UMUI_UNPAGE_SERIALNUMBER on macroline 7
Error in script "UninstSerialTest.nsi" on line 20 -- aborting creation process
```

**Comment by Nutzzz** [comment]
Tested UMUI_UNPAGE_ADDITIONALTASKS since it also has a function parameter and the same thing occurs.

**Comment by SuperPat45** [comment]
Thanks, fixed in c8233845f82131c6347409a216a49bbe6b5dbb0c

**Comment by Nutzzz** [comment]
Thank YOU!  👍
